README.md

```diff
@@ -1,5 +1,104 @@
-# Django ORM boilerplate for GitHub tasks
+# Tickets and orders
 
-- Warning: Use `pytest app` for testing - not simple `pytest`
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
\ No newline at end of file
+
+In `db/models.py` you already have tables you created earlier. Now
+you have to create tables:
+1. `Order`, with such fields:
+    - datetime field `created_at`, the datetime when the order was created,
+set `auto_add_now` to fill this field automatically during the creation.
+    - foreign key `user`, the user that created the order.
+   
+There should be implemented the string representation of the order:
+```python
+print(Order.objects.first())
+<Order: 2022-03-15 15:59:50.009379+00:00>
+```
+Orders should be ordered from the newest to the oldest by default.
+2. `Ticket`, with such fields:
+   - foreign key `movie_session`, is related to the table `MovieSession`
+   - foreign key `order`, is related to the table `Order`
+   - integer field `row`, the row of the seat 
+   - integer field `seat`, number of the seat in row
+
+There should be implemented the string representation of the ticket, show
+information about movie session, row and seat:
+```python
+print(Ticket.objects.first())
+<Ticket: Speed 2020-11-11 09:30:00+00:00 (row: 6, seat: 12)>
+```
+Ticket should implement method `clean()`. This method should check if 
+`row` and `seat` are correct and are not greater than values in 
+`ticket.movie_session.cinema_hall`, if they are - ValidationError should
+be raised.
+
+```python
+import datetime
+
+cinema_hall = CinemaHall.objects.create(name="Blue", rows=18, seats_in_row=24)
+movie_session = MovieSession.objects.create(
+    show_time=datetime.datetime(2022, 3, 20, 19, 30),
+    movie_id=1, 
+    cinema_hall=cinema_hall)
+order = Order.objects.create(user_id=1)
+
+ticket = Ticket.objects.create(
+    movie_session=movie_session,
+    order=order,
+    row=19,
+    seat=20
+)
+ticket.clean()
```

Comment:
Here the error should be during creation of ticket (if override `save` - this will happen)